code/modules/mob/living/carbon/xenomorph/update_icons.dm

```diff
@@ -150,6 +151,11 @@
 	icon = 'icons/Xeno/wound_overlays.dmi'
 	vis_flags = VIS_INHERIT_DIR
 
+/atom/movable/vis_obj/xeno_wounds/Initialize(mapload, datum/source)
+	. = ..()
+	if(isxenobehemoth(source))
+		icon = 'icons/Xeno/wound_overlays_3x3.dmi'
+
```

Comment:
Why not just make a castedatum var like we have for the wound icon state but use it for icon instead?

Reply:
Sure. Donezo.